brightener.cpp

```diff
@@ -16,38 +16,30 @@ shared_ptr<Image> BrightenWholeImage(shared_ptr<Image> inputImage, int& attenuat
     // For brightening, we add a certain grayscale (25) to every pixel.
```

Comment:
Suppose we want to avoid (accidental) brightening of an already brightened image. We may want to make these `Brighten...` functions take a `RawImage` instead of just `Image`. Try and see, is that possible to do?
Also - shouldn't the return type by `BrightenedImage`?

Reply:
Yup.. makes it tighter. Done